trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -118,12 +118,15 @@ static final byte FLAG_TRACED = 1; // 00000001
 boolean traced = (traceOptions & FLAG_TRACED) == FLAG_TRACED
 ```
 
-#### Traced Flag (00000001)
+#### Requested Flag (00000001)
 When set, the least significant bit recommends the request should be traced. A caller who
 defers a tracing decision leaves this flag unset.
```

Comment:
I would add something like this:

"An implementation might still decide on it's own if this flag is respected or not. Consider a scenario when capturing a trace induces cost. An upstream caller could set this flag to maliciously induce cost in a tracing system. It's recommended that an implementation is careful about this and only respects the flag if the upstream caller is trusted."